aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -1249,6 +1316,51 @@ private boolean s3KeyIdAccessKeyBothConfigured() {
     return (s3AccessKeyId == null) == (s3SecretAccessKey == null);
   }
 
+  private AwsCredentialsProvider credentialsProvider(String credentialsProviderClazz) {
```

Comment:
I would probably not call this "Clazz", we use Class everywhere else

Reply:
corrected.